lesson_07/homeworks_07.py

```diff
@@ -62,10 +93,49 @@ def find_substring(str1, str2):
 print(find_substring(str1, str2)) # поверне -1
 
 # task 7
+def is_even(numbers):
+    sum = 0
+    for i in numbers:
+        if i%2==0:
+            sum +=i
+    return sum
+
+numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+print(f"Total sum {is_even(numbers)}")
+
 # task 8
+def findStrElements(list):
+    lst2 =[]
+    for i in lst1:
+        if isinstance(i, str):
+            lst2.append(i)
+    return(lst2)
+
+lst1 = ['1', '2', 3, True, 'False', 5, '6', 7, 8, 'Python', 9, 0, 'Lorem Ipsum']
+print(findStrElements(lst1))
+
 # task 9
+def findLetter():
```

Comment:
```suggestion
def find_letter():
```
і_знову_ТРЕБА_так!!!